iOS/UI/Swipeable Input/SwipeableInputAccessoryMessageSender.swift

```diff
@@ -11,47 +11,27 @@ import Foundation
 protocol MessageSendingViewControllerType: UIViewController {
     func getCurrentMessageSequence() -> MessageSequence?
     func set(messageSequencePreparingToSend: MessageSequence?)
-    func sendMessage(_ message: Sendable)
-    func createNewConversation(_ sendable: Sendable)
+    func sendMessage(_ message: Sendable) async throws
 }
 
 protocol MessageSendingCollectionViewType: CollectionView {
     func getMessageDropZoneFrame(convertedTo view: UIView) -> CGRect
-    func getNewConversationContentOffset() -> CGPoint
 }
 
 class SwipeableInputAccessoryMessageSender: SwipeableInputAccessoryViewControllerDelegate {
-    
-    /// The type of message send method that the conversation VC is prepped for.
-    private enum SendMode {
-        /// The message will be sent to currently centered message.
-        case message
-        /// The message will the first in a new conversation.
-        case newConversation
-    }
 
     unowned let viewController: MessageSendingViewControllerType
     unowned let collectionView: MessageSendingCollectionViewType
-    let isConversationList: Bool
```

Comment:
I was able to delete a lot code in the message sender now that we don't create conversations with a swipe to the right.